api/libproject/devstart.rb

```diff
@@ -2637,7 +2642,7 @@ def deploy_tanagra(cmd_name, args)
 
   Dir.chdir('../tanagra-aou-utils/tanagra') do
     common.status "Building Tanagra API..."
-    common.run_inline("./gradlew -x test -PisMySQL clean service:build")
+    common.run_inline("GITHUB_ACTOR='dolbeew' GITHUB_TOKEN='#{op.opts.auth_token}' ./gradlew -x test -PisMySQL clean service:build")
```

Comment:
I guess we have to use someone's github user/token for this to work? Should we create a workbench user of some sort instead of using dolbeew?